Assets/MixedRealityToolkit-Examples/Demos/UX/Collections/Scenes/ObjectCollectionGallery.unity

```diff
@@ -4598,7 +4598,7 @@ MonoBehaviour:
 
     <size=75%>All Platforms</size>'
   m_isRightToLeft: 0
-  m_fontAsset: {fileID: 11400000, guid: 8f586378b4e144a9851e7b34d9b748ee, type: 2}
```

Comment:
Removed intentionally?

Reply:
As long as the text is still showing up, it should be fine then.

Reply:
I went ahead and removed them so there's no uncertainty.